From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Sat, 2 Apr 2011 16:43:33 +0200
Subject: [PATCH] create dbussystemservicedir

otherwise building fails if it doesn't exist:

( cd <DESTDIR>/usr/share/dbus-1/services && \
                rm -f org.freedesktop.systemd1.service && \
                ln -s ../system-services/org.freedesktop.systemd1.service org.freedesktop.systemd1.service )
/bin/sh: line 0: cd: <DESTDIR>/usr/share/dbus-1/services: No such file or directory

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 Makefile.am |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 873090e..e4cf1f5 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1340,6 +1340,7 @@ endif
 		$(DESTDIR)$(pkgsysconfdir)/system/multi-user.target.wants \
 		$(DESTDIR)$(pkgsysconfdir)/system/getty.target.wants \
 		$(DESTDIR)$(pkgsysconfdir)/user \
+		$(DESTDIR)$(dbussessionservicedir) \
 		$(DESTDIR)$(sysconfdir)/xdg/systemd
 	( cd $(DESTDIR)$(sysconfdir)/xdg/systemd/ && \
 		rm -f user && \
-- 
1.7.4.1

